scripts/zones/Halvung/npcs/qm5.lua

```diff
@@ -0,0 +1,32 @@
+-----------------------------------
+-- Area: Halvung
+--  NPC: ??? (Bracelet of verve)
+--  Trade Moblin Oil
+-----------------------------------
+local ID = require("scripts/zones/Halvung/IDs")
+require("scripts/globals/npc_util")
+require("scripts/globals/keyitems")
+
```

Comment:
Ugh, I'm terribly sorry. I just noticed you are missing a require here.
Please add this.

`require("scripts/globals/items")`

This is needed for the xi.item.MOBLIN_OIL bit to work.

Reply:
Weird, I loaded it up on test server deleted KI and traded and it worked.... will put in. Thanks again!

Reply:
That can happen at times, but it's just good practice to add it to any script that uses it

Reply:
For sure. Sorry I should have done better to start. Still trying to grasp all the changes ha. Now I gotta figure out the doors - should I keep adding to here or make separate?

Reply:
I would make it separate. Smaller = easier to review.

Reply:
> Weird, I loaded it up on test server deleted KI and traded and it worked.... will put in. Thanks again!

Its because if another script happened to load that global into memory, it will still "know" of it. But we can't rely on that so we require it every time we're going to use something form it. If its already loaded when the code hits that require it's fine, if not then now it is. Lua is funny that way.